src/client/client4.js

```diff
@@ -822,6 +830,17 @@ export default class Client4 {
         );
     };
 
+    updateTeamScheme = async (teamId, schemeId) => {
+        const patch = {scheme_id: schemeId};
+
+        this.trackEvent('api', 'api_teams_update_scheme', {team_id: teamId, ...patch});
```

Comment:
I think the diagnostics should be explicit in what it tracks, otherwise this could potentially in future end up accidentally leaking data that is inappropriate to track.

Reply:
@grundleborg Can you be more specific?

Reply:
Sorry, didn't explain that at all clearly.

The `...patch` in a call to `trackEvent()` worries me slightly, because if an extra field were to be added to that `patch` object that it isn't appropriate to send back in the telemetry data, it would get sent unless the person adding it remembered to update these `trackEvent` calls. I know it's unlikely, but where telemetry is concerned, I'd prefer to be explicit about which fields are getting sent.

Reply:
So your concern is the use of the spread operator? You know it's defined explicitly in the line above right? It's not some magic parameter.

Reply:
Oh yeah, I missed that (assumed it was a parameter being passed into the function). In which case, no objection here.